cmd/copy.go

```diff
@@ -453,7 +460,7 @@ func (raw rawCopyCmdArgs) cookWithId(jobId common.JobID) (cookedCopyCmdArgs, err
 			cooked.pageBlobTier != common.EPageBlobTier.None() {
 			return cooked, fmt.Errorf("blob-tier is not supported while uploading to ADLS Gen 2")
 		}
-		if cooked.preserveNTFSACLs {
+		if cooked.preserveSMBPermissions {
```

Comment:
Next line needs NTFS replaced with SMB. Can you also to a search for NTFS to see if any others have been missed?